### DISCUSSION TITLE
duckdb

### DISCUSSION THREAD

**Discussion by jessekrubin** [ryanfowler/async-sqlite#17] [Show and tell]
I forked and re-jiggered your repo to support `duckdb` (`async-duckdb`) -- it is a private repo (for now) but I was wondering if I could get your thoughts before publishing/making public.

There were only some changes I had to make, but overall it was a lot of `%s/rusqlite/duckdb/g` (and related find-replaces).

I sent you an invite to the repo!

**Comment by ryanfowler** [comment]
Oh cool! Sounds good, I'll be travelling for a bit so may not be too responsive during that time...
